english/forest/fire/woods/follow/jumpInWaterFall/jump.md

```diff
@@ -0,0 +1,6 @@
+You fall and keep falling until you hit the water hard. You feel pain as you have never experienced before.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.